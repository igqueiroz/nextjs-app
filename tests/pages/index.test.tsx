import React from 'react'
import {render} from '@testing-library/react'
import Home from 'pages/index'

describe('Index page', () => {
  it('should be defined', () => {
    const {container} = render(<Home />)
    expect(container).toMatchSnapshot()
  })
})
