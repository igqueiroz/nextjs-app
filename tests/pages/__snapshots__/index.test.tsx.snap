// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page should be defined 1`] = `
.c0 {
  color: red;
  font-size: 20px;
}

<div>
  <div>
    <h1
      class="c0"
    >
      Agenda 4 Pets
    </h1>
  </div>
</div>
`;
